roles/oVirt.datacenter-cleanup/tasks/vms.yml

```diff
@@ -4,11 +4,13 @@
     auth: "{{ ovirt_auth }}"
     pattern: "datacenter={{ data_center_name }}"
 
-- name: Remove VMs
+- name: Remove all VMs apart from HE one if exists
   ovirt_vms:
     state: absent
     id: "{{ ovirt_item.id }}"
     auth: "{{ ovirt_auth }}"
   with_items: "{{ ovirt_vms }}"
+  when: not hosted_engine or
+        (ovirt_item.name != hosted_engine_vm_name)
```

Comment:
I would keep condition on one line rather then brake the line this way

Reply:
fixed